docs/src/security/gdpr.md

```diff
@@ -9,18 +9,39 @@ description: |
 
 {{< description >}}
 
+# Measures
+
 As part of our measures we have implemented the following:
 
 * **Data Protection Officer**: Appointment of a Security Officer who also holds the Data Protection Officer (DPO) role.
-* **Data Breach Policy**: We have updated our data breach policy and procedures, and have confirmed that all of our suppliers are compliant with breach notification requirements.
-* **Consent**: We have confirmed that all of our customer communication, both business and marketing-related, is opt-in, and we do not collect customers' information without their consent.
-* **Data Governance**: We have completed internal audits on all of our suppliers to ensure their internal security measures are adequate, and can confirm that our in-scope suppliers are GDPR compliant.
-* **Data Protection by design**: We have implemented company policies to ensure that all of our employees receive the necessary compliance training and follow proper protocols regarding security. Further, privacy and data protection implications are assessed at the start of every new project. 
+* **Data Breach Policy**: We updated our data breach policy and procedures, and confirmed that all of our suppliers are compliant with breach notification requirements.
+* **Consent**: We confirmed that all of our customer communication, both business and marketing-related, is opt-in, and we do not collect customers' information without their consent.
+* **Data Governance**: All vendors undergo a compliance review and a security review. We utilize DPAs and SCCs with all vendors that process Personal Data. We do not use Privacy Shield.
```

Comment:
That'd probably be easier to understand if we don't use the acronyms directly, as the terms haven't been explained at this point.
Also, "personal data" appears non-capitalised somewhere else in the page.

```suggestion
* **Data Governance**: All vendors undergo a compliance review and a security review. We utilize Data Processing Agreements (DPAs) and Standard Contractual Clauses (SCCs) with all vendors that process personal data. We do not use Privacy Shield.
```